kv/dist_sender.go

```diff
@@ -867,7 +865,10 @@ func (ds *DistSender) sendChunk(ctx context.Context, ba roachpb.BatchRequest) (*
 				}
 				boundedArg, ok := args.(roachpb.Bounded)
 				if !ok {
-					// Non-bounded request. We will have to query all ranges.
+					// Non-bounded request. We will have to continue querying
+					// until this request is satisfied. This request might
+					// have already been saturated at this stage but we have
```

Comment:
I would reserve "saturated" to mean "reached the number of requested entries". You're using it for "the key range has been completely queried".